examples/siro_sandbox/sandbox_app.py

```diff
@@ -709,11 +748,10 @@ def sim_update(self, dt):
                 not self.env._sim.navmesh_visualization  # type: ignore
             )
 
-        action = self.ctrl_helper.update(self.obs)
+        action = self.ctrl_helper.update(self._obs)
+        self.env_step(action)
 
-        self.obs = self.env.step(action)
-
-        post_sim_update_dict = {}
+        post_sim_update_dict = {"should_exit": should_exit}
```

Comment:
I don't see a reason to send the message at all if the value here would be False.